SPDLOG_QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,90 @@
+This document is a declaration of software quality for the `spdlog` external dependency imported by the `spdlog_vendor` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `spdlog` Quality Declaration
+
+TODO: Declare the quality level for `spdlog`
+
+## Version Policy
+
+### Version Scheme
+
+There is a three-part version attached to `spdlog` releases.
+The sources define the version parts as `MAJOR`.`MINOR`.`PATCH`.
+There is an old ticket where the project's primary developer discusses versioning, but there is no documentation officially stating what approach the developer is taking.
+
+### API/ABI Stability Within a Released ROS Distribution
+
+As an external package, `spdlog` is not released on the same cadance as ROS.
+However, the `spdlog_vendor` package controls what version of `spdlog` is acceptable, so API/ABI stability within a ROS release can be enforced.
+
+### Public API Declaration
+
+Because `spdlog` can be consumed as a header-only library, nearly all of the source code can be considered part of the public API.
+For the compiled portion of `spdlog`, the `SPDLOG_API` C++ macro indicates the symbols that are part of the public API.
+
+## Change Control Process
+
+The source code repository for `spdlog` is hosted in Github, and the change process follows a typical Github pull request model.
+It appears that a single developer is reviewing and merging the proposed changes.
+The developer appears to be active in triaging incoming issues and addressing incoming pull requests in a timely manner.
+
+Incoming pull requests automatically trigger continuous integration testing across Linux and macOS, and must run successfully to be merged.
```

Comment:
Note the specific platforms:

* Ubuntu 16.04 and 18.04
* OS X Xcode-9.4.1 (I couldn't find the os version)
* Windows VS 2015 and 2017 (Couldn't find windows version)

Reply:
Given that this package has no external dependencies, I'm not sure how useful the OS version information is. Additionally, it doesn't look like the package's build configurations consistently dictate what versions to use - it sometimes seems to be the default that the hosting infrastructure uses. Seems like pretty fragile information to include, but I'll put it in.